java/src/main/java/com/start/Connection.java

```diff
@@ -1,19 +1,30 @@
 package com.start;
 
-import com.mongodb.client.MongoClient;
-import com.mongodb.client.MongoClients;
 import org.bson.Document;
 
-import java.util.ArrayList;
-import java.util.List;
+import com.mongodb.ConnectionString;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.MongoException;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoDatabase;
 
 public class Connection {
 
     public static void main(String[] args) {
-        String connectionString = "mongodb://localhost:27017";
-        try (MongoClient mongoClient = MongoClients.create(connectionString)) {
-            List<Document> databases = mongoClient.listDatabases().into(new ArrayList<>());
-            databases.forEach(db -> System.out.println(db.toJson()));
+        for (String connectionUri : args) {
+            try (MongoClient mongoClient = MongoClients.create(connectionUri)) {
+                MongoDatabase database = mongoClient.getDatabase("test");
+
+                try {
+                    Document command = new Document("ping", 1);
+                    Document res = database.runCommand(command);
+                    System.out.println(res);
+                } catch (MongoException e) {
+                    System.err.println("An error occurred: " + e);
+                }
```

Comment:
That would cause this program to exit with code 0, passing the dance test.

I think we should not catch this exception